06.test-ismb-data.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import utils
+import pickle
+import argparse
+import requests
+import numpy as np
+import pandas as pd
+from keras.models import load_model
+from keras.preprocessing import sequence
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-l", "--ngrams", type=int, default=3)
+parser.add_argument("-m", "--model_name", default="LSTM")
+args = parser.parse_args()
+
+# Load model
+model = load_model("models/%s.h5" % args.model_name)
+filename = "models/" + args.model_name + "_idx_dic.pkl"
+idx_dic = pickle.load(open(filename, "rb"))
+categories = pd.read_csv("models/%s_categories.txt" % args.model_name, header=None)
+
+# Get PubMed author data from iscb-diversity repo
+df = pd.read_csv("https://github.com/greenelab/iscb-diversity/raw/master/data/pubmed/authors.tsv.xz", sep='\t')
```

Comment:
Would be best to let the user specify the `greenelab/iscb-diversity` commit hash such that we can reproducible run this repo. Perhaps you could create a config.py file (or in utils.py) that has a variable like `iscb_diversity_commit="3f0415686c4d59f423852679258984c63754e038"`.

Then replace `master` with the commit in the above URL.